pkg/cli/featureflag.go

```diff
@@ -0,0 +1,21 @@
+package cli
+
+import (
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+const (
+	//RequiresAccessCode flag
+	RequiresAccessCode string = "requires-access-code"
```

Comment:
My preference here would be to use the name space `feature-flag`. So this:
 
```suggestion
	FeatureFlagAccessCode string = "feature-flag-access-code"
```